docs/serving/samples/cloudevents/cloudevents-nodejs/index.js

```diff
@@ -0,0 +1,87 @@
+const express = require('express')
+const { HTTPReceiver } = require('cloudevents-sdk')
+const { CloudEvent, StructuredHTTPEmitter } = require('cloudevents-sdk/v1')
+const Constants = require('cloudevents-sdk/lib/bindings/http/constants.js')
```

Comment:
Minor nit, but you could destructure this as you are doing with `{ CloudEvent, StructuredHTTPEmitter }` above. E.g. 

```js
const { BINARY_HEADERS_1 } = require('cloudevents-sdk/lib/bindings/http/constants.js');
```